difference-of-squares/difference-of-squares.rb

```diff
@@ -0,0 +1,26 @@
+class Squares
+  def initialize(number)
+	   @@sumOfSquares = 0
+	   @@squareOfSum = 0
+	   @@diff_Num = 0
+    while number > 0
+		    @@squareOfSum = @@squareOfSum + number
+		    @@sumOfSquares  = @@sumOfSquares + (number * number)
+	      number -= 1
+    end
+		@@squareOfSum = (@@squareOfSum * @@squareOfSum)
+	  @@diff_Num = (@@squareOfSum - @@sumOfSquares)
+	end
+
+	def square_of_sum
+		return @@squareOfSum
+	end
+
+	def sum_of_squares
+		return @@sumOfSquares
+	end
+
+	def difference
+		return @@diff_Num
+	end
+end
```

Comment:
fix indentation in whole file and why you have used class variables? Please remove it and use instance variables